### PR TITLE
Allow signing arbitrary texts in key details dialog

### PR DESCRIPTION
**Pull request by alexfmpe**

### CODE REVIEW

File: backend/sass/common.blocks/_wallet.scss [hunk 1]
```diff
@@ -83,3 +83,27 @@
...
+  justify-content: space-between;
+
+  background: $chainweaver-blue;
+  color: white;
+
+  .copy-status {
+    transition: opacity, transform 0.25s linear;
+  }
+
+  .copy-success {
```
Comment (mankyKitty) [inline backend/sass/common.blocks/_wallet.scss]: Should this include the `&` : `&.copy-success` ?

Comment (alexfmpe) [reply backend/sass/common.blocks/_wallet.scss]: Hmm not like this, but it should if they're nested inside `.copy-status` like the setup ones are.
I'll nest here as well

File: frontend/src/Frontend/UI/Button.hs [hunk 2]
```diff
@@ -192,6 +196,27 @@ copyButton cfg t = do
     _ <- copyToClipboard $ tag t onClick
     pure onClick
 
+copyButtonStatus :: (DomBuilder t m, MonadFix m, MonadHold t m, MonadJSM (Performable m), PerformEvent t m, PostBuild t m)
+                 => UiButtonCfg
```
Comment (tomsmalley) [inline frontend/src/Frontend/UI/Button.hs]: cursed indentation

File: frontend/src/Frontend/UI/Dialogs/KeyDetails.hs [hunk 3]
```diff
@@ -62,15 +69,50 @@ uiKeyDetailsDetails keyIndex key onClose onCloseExternal = Workflow $ do
...
-      fmap value $ mkLabeledClsInput False "Notes" $ \cls -> uiInputElement $ def
+      notes <- fmap value $ mkLabeledClsInput False "Notes" $ \cls -> uiInputElement $ def
         & inputElementConfig_initialValue .~ unAccountNotes (_key_notes key)
         & initialAttributes . at "class" %~ pure . maybe (renderClass cls) (mappend (" " <> renderClass cls))
 
+      void $ accordionItemWithClick False mempty (accordionHeaderBtn "Advanced") $ withSecretKey $ \pk -> do
+        txt <- fmap value $ mkLabeledClsInput False "Text to sign" $ \cls -> uiTextAreaElement $ def
+          & initialAttributes .~ "class" =: renderClass cls
+
+        sigEv <- do
```
Comment (tomsmalley) [inline frontend/src/Frontend/UI/Dialogs/KeyDetails.hs]: this do block seems pointless

Comment (alexfmpe) [reply frontend/src/Frontend/UI/Dialogs/KeyDetails.hs]: Ah yes

File: frontend/src/Frontend/UI/Button.hs (hunk 2 above)
Comment (alexfmpe) [reply frontend/src/Frontend/UI/Button.hs]: Thank you for the exorcism

**Pull request merged**